tool/simulate-dialogs.ts

```diff
@@ -75,36 +77,104 @@ export function initArgparse(subparsers : argparse.SubParser) {
         type: fs.createReadStream,
         help: 'Input dialog file'
     });
+    parser.add_argument('--nlu-server', {
+        required: false,
+        help: `The URL of the natural language server to parse user utterances. Use a file:// URL pointing to a model directory to use a local instance of genienlp.
+               If provided, will be used to parse the last user utterance instead of reading the parse from input_file.`
+    });
+    parser.add_argument('--output_mistakes_only', {
+        action: 'store_true',
+        help: 'If set and --nlu-server is provided, will only output partial dialogues where a parsing mistake happens.',
+        default: true
```

Comment:
If you set default: true, then you need the negated version of this option, otherwise there is no way to unset it.